### PR TITLE
Feature: add possibility to automatically create repository in ECR

### LINKED ISSUE #5: Create repository in ECR automatically
It would be nice if this plugin would create repository in ECR if it doesn't exists. Possibly as an option.

### PR DESCRIPTION
**Pull request by gregory90**
Fixes #5 .
Let me know if I should change anything.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -89,6 +89,9 @@ func main() {
 	username := authTokens[0]
 	password := authTokens[1]
 	registry := registryURL.Host
+
+	shortRepoName := vargs.Repo
```
Comment (tboerger) [inline main.go:93]: No need for this variable as it gets accessed anyway only once.

Comment (gregory90) [inline main.go:93]: It's needed because of https://github.com/gregory90/drone-ecr/blob/master/main.go#L99

**Comment by tboerger** [comment]
LGTM

**Comment by jackspirou** [comment]
LGTM

**Comment by gregory90** [comment]
Thanks! Shouldn't PR merge automatically when 2 LGTMs are in the comments?:)

**Pull request merged**